### PR TITLE
Book suggestions specs

### PR DESCRIPTION
**Pull request by jhonmunoz**
`book_suggestion` controller spec implemented

### CODE REVIEW

File: spec/controllers/api/v1/book_suggestions_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe Api::V1::BookSuggestionsController, type: :controller do
+  describe 'POST #create' do
+    context 'when creating a valid book suggestion' do
+      let(:valid_params) do
+        {
+          book_suggestion: {
```
Comment (hdf1996) [inline spec/controllers/api/v1/book_suggestions_controller_spec.rb]: You can use `attributes_for` from factory bot here

File: spec/controllers/api/v1/book_suggestions_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+            year:    2010
+          }
+        }
+      end
+      it 'creates a new suggestion' do
+        expect do
+          post :create, params: valid_params
+        end.to change { BookSuggestion.count }.by(1)
+      end
+      it 'responds with 201 status' do
```
Comment (hdf1996) [inline spec/controllers/api/v1/book_suggestions_controller_spec.rb]: keep an empty line before every it

File: spec/controllers/api/v1/book_suggestions_controller_spec.rb [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        post :create, params: valid_params
+        expect(response).to have_http_status(:created)
+      end
+    end
+
+    context 'when creating an invalid book suggestion' do
+      let(:invalid_params) do
+        {
```
Comment (hdf1996) [inline spec/controllers/api/v1/book_suggestions_controller_spec.rb]: You can use attributes_for(:book_suggestion, author: nil) for example

**Review by hdf1996 — CHANGES_REQUESTED** [review]

File: spec/controllers/api/v1/book_suggestions_controller_spec.rb (hunk 2 above)
Comment (jhonmunoz) [reply spec/controllers/api/v1/book_suggestions_controller_spec.rb]: done

File: spec/controllers/api/v1/book_suggestions_controller_spec.rb (hunk 1 above)
Comment (jhonmunoz) [reply spec/controllers/api/v1/book_suggestions_controller_spec.rb]: A much better approach

File: spec/controllers/api/v1/book_suggestions_controller_spec.rb (hunk 3 above)
Comment (jhonmunoz) [reply spec/controllers/api/v1/book_suggestions_controller_spec.rb]: Perfect!

**Review by hdf1996 — APPROVED** [review]

**Review by hdf1996 — APPROVED** [review]

**Pull request merged**